src/runtime/README.md

```diff
@@ -23,66 +29,62 @@ The runtime is
  allowing it
 to work seamlessly with both Docker and Kubernetes respectively.
 
-## License
+## Download and install
 
-The code is licensed under an Apache 2.0 license.
-See [the license file](https://github.com/kata-containers/kata-containers/blob/main/LICENSE) for further details.
+[![Get it from the Snap Store](https://snapcraft.io/static/images/badges/en/snap-store-black.svg)](https://snapcraft.io/kata-containers)
 
-## Platform support
+See the [installation guides](https://github.com/kata-containers/kata-containers/blob/main/docs/install/README.md)
+available for various operating systems.
 
-Kata Containers currently works on systems supporting the following
-technologies:
+## Architecture overview
 
-- [Intel](https://www.intel.com) VT-x technology.
-- [ARM](https://www.arm.com) Hyp mode (virtualization extension).
-- [IBM](https://www.ibm.com) Power Systems.
-- [IBM](https://www.ibm.com) Z mainframes.
-### Hardware requirements
+See the [architecture overview](../../docs/design/architecture)
+for details on the Kata Containers design.
 
-The runtime has a built-in command to determine if your host system is capable
-of running and creating a Kata Container:
+## Configuration
 
-```bash
-$ kata-runtime check
-```
+The runtime uses a TOML format configuration file called `configuration.toml`.
+The file is divided into sections for settings related to various
+parts of the system including the runtime itself, the [agent](../agent) and
+the [hypervisor](#hypervisor-specific-configuration).
+
+Each option has a comment explaining it use.
 
 > **Note:**
 >
-> - By default, only a brief success / failure message is printed.
-> If more details are needed, the `--verbose` flag can be used to display the
-> list of all the checks performed.
->
-> - `root` permission is needed to check if the system is capable of running
-> Kata containers. In this case, additional checks are performed (e.g., if another
-> incompatible hypervisor is running).
+> - The initial values in the configuration file provide a good default configuration.
+>   You may need to modify this file to optimise or tailor your system, or if you have
+>   specific requirements.
 
-## Download and install
+### Configuration file location
 
-[![Get it from the Snap Store](https://snapcraft.io/static/images/badges/en/snap-store-black.svg)](https://snapcraft.io/kata-containers)
+#### Runtime configuration file location
 
-See the [installation guides](https://github.com/kata-containers/kata-containers/blob/main/docs/install/README.md)
-available for various operating systems.
+The shimv2 runtime looks for its configuration in the following places (in order):
 
-## Quick start for developers
+1. The `io.data containers.config.config_path` annotation specified
+   in the OCI configuration file (`config.json` file) used to create the pod sandbox.
 
-See the
-[developer guide](../../docs/Developer-Guide.md).
+1. The containerd
```

Comment:
I see the merit of making patches cleaner by always having number 1. However, I think this makes it hard to read as plain text, which is a mission statement of Markdown IIRC.

Reply:
Fixed.